src/lib/style/components/_download-cart-page.scss

```diff
@@ -92,6 +92,12 @@
       display: flex;
       column-gap: 10px;
       >* {
+        &:first-child {
```

Comment:
The first item (leftmost) should not have padding on the left (it's now in a container), and the last item (rightmost) should not have padding on the right.